### ISSUE TITLE
VSCode Insiders fails to update

### ISSUE THREAD

**Issue by NotMyself** [microsoft/winget-cli#4638]
### Brief description of your issue

I have installed vscode insiders version using winget. The built in self update feature fails.

### Steps to reproduce

1. `winget install Microsoft.VisualStudioCode.Insiders`
2. Wait a day or so for a new version to be published.
3. Launch VSCode Insiders.
4. When notified of an update, select Install Update.
5. Select Update Now.

### Expected behavior

VSCode is successfully updated.

### Actual behavior

Two errors:
![Screenshot 2024-07-15 135338](https://github.com/user-attachments/assets/0e522e9d-4cb3-41e0-86fd-a4094a590bf7)
![Screenshot 2024-07-15 135415](https://github.com/user-attachments/assets/1e7116f1-052a-4410-8a1e-ab0bf5117c3f)

Here is the install log:
```
Jul 15 18:01:24.253 INFO Starting: C:\Program Files\Microsoft VS Code Insiders\Code - Insiders.exe, false, Updating Visual Studio Code...
Jul 15 18:01:24.253 ERRO Bad arguments: Code path doesn't seem to exist: C:\Program Files\Microsoft VS Code Insiders\Code - Insiders.exe
```

### Environment

```shell
[winget --info]
Windows: Windows.Desktop v10.0.19045.4529
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.23.1911.0
```

**Comment by Trenly** [comment]
@lszomoru - is this perhaps an issue with the VS Code installer itself?

**Comment by lszomoru** [comment]
@NotMyself, this does not seem to be related to `winget` as VS Code can be installed through `winget` but updating VS Code is being handled by VS Code itself. What is the version of VS Code that you have installed that fails to upgrade? Thank you!

**Comment by NotMyself** [comment]
@trenly & @lszomoru  - I waffled on where to report for sure. I decided here because the it appears winget is installing the app in a way that breaks the app's auto update feature. I currently have 1.92.0-insiders installed. Here is the rest of the about screen in case it might be helpful.

![Screenshot 2024-07-16 080350](https://github.com/user-attachments/assets/6af71f7b-0716-48f7-ac79-6848c3f33674)

**Comment by Trenly** [comment]
> it appears winget is installing the app in a way that breaks the app's auto update feature.

All WinGet does is download the latest installer file and run it with the command line switches to make it install unattended. I expect you'd likely see the same issue if you downloaded and ran the installer yourself

**Comment by NotMyself** [comment]
```
you'd likely see the same issue if you downloaded and ran the installer yourself
```

Fair enough, seem like a lot of people would be having this problem if that were the case. I suspect there is a step missing somewhere. If I downloaded the installer and installed the package in a non-standard location, I would expect it to make an environmental change that the update could reference to find that location be it envvar or registry entry.

So I guess either the package manifest for the package on winget needs to do that, or the vscode installer needs to do it.  I can submit a ticket to the vscode repo.

**Comment by NotMyself** [comment]
It looks like this issue is resolved here microsoft/vscode#220999
